go/vt/vttablet/tabletmanager/vreplication/engine.go

```diff
@@ -102,37 +113,41 @@ func (vre *Engine) Open(ctx context.Context) error {
 
 // executeFetchMaybeCreateTable calls DBClient.ExecuteFetch and does one retry if
 // there's a failure due to mysql.ERNoSuchTable or mysql.ERBadDb which can be fixed
-// by re-creating the _vt.vreplication table.
+// by re-creating the vreplication tables.
 func (vre *Engine) executeFetchMaybeCreateTable(dbClient binlogplayer.DBClient, query string, maxrows int) (qr *sqltypes.Result, err error) {
 	qr, err = dbClient.ExecuteFetch(query, maxrows)
 
 	if err == nil {
 		return
 	}
 
-	// If it's a bad table or db, it could be because _vt.vreplication wasn't created.
-	// In that case we can try creating it again.
+	// If it's a bad table or db, it could be because the vreplication tables weren't created.
+	// In that case we can try creating them again.
 	merr, isSQLErr := err.(*mysql.SQLError)
 	if !isSQLErr || !(merr.Num == mysql.ERNoSuchTable || merr.Num == mysql.ERBadDb || merr.Num == mysql.ERBadFieldError) {
 		return qr, err
 	}
 
-	log.Info("Looks like _vt.vreplication table may not exist. Trying to recreate... ")
+	log.Info("Looks like the vreplcation tables may not exist. Trying to recreate... ")
```

Comment:
I removed the confusing comment a few weeks ago. It was in tablet manager. This should not spam and looks useful, because it will let us know when the tables got created.